test/pages/index.spec.js

```diff
@@ -8,6 +8,6 @@ describe("index", () => {
     render(<Home />);
     const expected =
       "Instantly deploy your Next.js site to a public URL with Vercel.";
-    expect(screen.getByText(expected)).toBeDefined();
+    expect(screen.queryByText(expected)).not.toBeNull();
```

Comment:
Just FYI: the standard assertion for react testing library stuff existing on your screen is `.toBeInTheDocument()`

Reply:
Sounds good, I switched over to that after installing/configuring the jest-dom helpers.